src/pudl/glue/ferc1_eia.py

```diff
@@ -76,406 +78,230 @@ def get_plant_map() -> pd.DataFrame:
+    """Grab the utlity ids (reported as `entity_id`) in the FERC1 XBRL database."""
+    all_utils_ferc1_xbrl = (
+        pd.read_sql(
+            "SELECT entity_id, respondent_legal_name FROM identification_001_duration",
+            ferc1_xbrl_engine,
+        )
+        .rename(
+            columns={
+                "entity_id": "utility_id_ferc1_xbrl",
+                "respondent_legal_name": "utility_name_ferc1",
+            }
+        )
+        .drop_duplicates(subset=["utility_id_ferc1_xbrl"])
+    )
+    return all_utils_ferc1_xbrl
 
-    Args:
-        None
 
-    Returns:
-        A DataFrame with three columns: ``plant_name``, ``utility_id_ferc1``, and
-        ``utility_name_ferc1``. Each row represents a unique combination of
-        ``utility_id_ferc1`` and ``plant_name``.
-    """
-    # If we're only trying to get the NEW plants, then we need to see which
-    # ones we have already integrated into the PUDL database. However, because
-    # FERC doesn't use the same plant names from year to year, we have to rely
-    # on the full mapping of FERC plant names to PUDL IDs, which only exists
-    # in the ID mapping spreadhseet (the FERC Plant names in the PUDL DB are
-    # canonincal names we've chosen to represent all the varied plant names
-    # that exist in the raw FERC DB.
-    ferc1_mapped_plants = (
-        get_plant_map()
-        .loc[:, ["utility_id_ferc1", "utility_name_ferc1", "plant_name_ferc1"]]
-        .dropna(subset=["utility_id_ferc1"])
-        .pipe(
-            pudl.helpers.simplify_strings,
-            columns=["utility_id_ferc1", "utility_name_ferc1", "plant_name_ferc1"],
+def get_utils_ferc1_raw_dbf(ferc1_engine_dbf: sa.engine.Engine) -> pd.DataFrame:
+    """Grab the utlity ids (reported as `respondent_id`) in the FERC1 DBF database."""
+    all_utils_ferc1_dbf = (
+        pd.read_sql_table("f1_respondent_id", ferc1_engine_dbf)
+        .rename(
+            columns={
+                "respondent_id": "utility_id_ferc1_dbf",
+                "respondent_name": "utility_name_ferc1",
+            }
         )
-        .astype({"utility_id_ferc1": int})
-        .drop_duplicates(["utility_id_ferc1", "plant_name_ferc1"])
-        .sort_values(["utility_id_ferc1", "plant_name_ferc1"])
+        .pipe(pudl.helpers.simplify_strings, ["utility_name_ferc1"])
+        .drop_duplicates(subset=["utility_id_ferc1_dbf"])
     )
-    return ferc1_mapped_plants
+    return all_utils_ferc1_dbf
 
 
-def get_mapped_utils_ferc1():
-    """Read in the list of manually mapped utilities for FERC Form 1.
+class GenericPlantFerc1TableTransformer(Ferc1AbstractTableTransformer):
+    """Generic plant table transformer.
 
-    Unless a new utility has appeared in the database, this should be identical
-    to the full list of utilities available in the FERC Form 1 database.
+    Intended for use in compiling all plant names for mannual ID mapping.
```

Comment:
mannual -> manual

Reply:
Some more explanation of what this class is doing would be helpful. And maybe a less generic name? `PlantsFerc1Preprocessor`?

Reply:
wow english is weird i truly though this had two "n"'s... fixed it in several places

Reply:
Well. It does when it's annual. But not manual.